jobs/myP2P3Links.js

```diff
@@ -9,48 +9,78 @@ module.exports = {
     'jira'
   ],
   userInfoNeeded: [
-    'jiraUserId'
+    'jiraUserId',
+    'jiraTeam'
   ],
   hiddenFromHelp: false,
   phrases: [
     'p2 links',
     'p3 links',
+    'give me my team\'s p2 links',
     'give me p2 links',
-    'give me p3 links'
+    'give me p3 links',
+    'give me my p2 details',
+    'give me details for my p3s',
+    'give me the details for the team\'s p3 bugs'
   ],
   fn: function ({
                   bot,
                   message,
                   jira,
-                  phraseMatch,
                   userInfo
                 }) {
-    let filterPath;
-    let priorityString;
-    if (phraseMatch.includes('p2')) {
-      filterPath = 'filter/15209';
-      priorityString = 'P2';
-    } else {
-      filterPath = 'filter/17400';
-      priorityString = 'P3'
+    const filters = [];
+    let assignee = `assignee = ${ userInfo.jiraUserId } AND `;
+    let ownershipString = 'Your';
+    let isTeam = false;
+    let isMine = false;
+    const p2p3 = [];
+    const messageText = message.text.toLowerCase();
+    if (messageText.includes('team')) {
+      isTeam = true}
+    if (messageText.includes('my') || messageText.includes('mine')) {
+      isMine = true}
+    if (isTeam && isMine) {
+      assignee = `assignee in membersOf(${ userInfo.jiraTeam }) AND `;
+      ownershipString = 'Your team\'s';
     }
-    jira.get(filterPath).
-    then(filterResults => {
-      console.log(filterResults);
-      utils.listIssuesInResult({
-        jqlOrPromise: `assignee = ${userInfo.jiraUserId} AND ` +filterResults.data.jql,
-        bot: bot,
-        message: message,
-        jira: jira
-      }).
-      then(issueList => {
-        bot.reply(message, `*Your ${priorityString} bugs are*:\n${issueList}`);
-      }).catch(err => {
-        console.log('$$$$$$$$');
-      });
+    else if (isTeam) {
```

Comment:
This is pretty similar to what is in myOpenBugs.js, we should try to find a way to add team into the dependencies, or add a utility function to get it, and maybe also assignee.